### ISSUE TITLE
False positive from hugged values in dict literals

### ISSUE THREAD

**Issue by PeterJCLaw** [PeterJCLaw/flake8-balanced-wrapping#1]
``` python
        league_matches = [{'A': Match(
            0,
            "Match 0",
            'A',
            [],
            datetime(2014, 4, 27, 12, 30),
            datetime(2014, 4, 27, 12, 35),
            MatchType.league,
            use_resolved_ranking=False,
        )}]
```
Yields:
```
1:27: BWR001 Dict is wrapped badly - 2 elements on the same line
```

There's probably an argument to be had about whether this particular case is actually clear, however the general case is probably something we want to allow.

**Comment by PeterJCLaw** [comment]
A more compelling example of this may be:
``` python
def foo():
    return LayoutData({'teams': [
        RegionData({
            'name': RegionName('a-group'),
            'display_name': "A group",
            'description': '',
            'teams': [TLA('ABC'), TLA('DEF'), TLA('GHI')],
        }),
        RegionData({
            'name': RegionName('b-group'),
            'display_name': "B group",
            'description': '',
            'teams': [TLA('JKL'), TLA('MNO'), TLA('PQR')],
        }),
    ]})
```
```
2:23: BWR001 Dict is wrapped badly - 2 elements on the same line
```

**Comment by PeterJCLaw** [comment]
Fixed by e395552ae8665081f215bfdcdf574a785e1efbc3.
